fedmsg_meta_fedora_infrastructure/buildsys.py

```diff
@@ -60,6 +60,10 @@ def subtitle(self, msg, **config):
             tmpl = self._(
                 "Package list change for {package}:  '{tag}'{inst}")
             return tmpl.format(inst=inst, **msg['msg'])
+        elif 'buildsys.rpm.sign' in msg['topic']:
+            tmpl = self._('Koji imported a gpg signature for '
```

Comment:
Ah, but koji didn't sign the package -- someone else did (this is usually sigul in our case).